### PR TITLE
Switch default zdc to baseline

### PR DESCRIPTION
**Pull request by ajentsch**
### Briefly, what does this PR introduce?

This PR changes the default ZDC to the project baseline.

### What kind of change does this PR introduce?
- [ ] Bug fix (issue #__)
- [ ] New feature (issue #__)
- [ ] Documentation update
- [ x] Other:  Changes default compact file for the ZDC.

### Please check if this PR fulfills the following:
- [ x] Tests for the changes have been added
- [x ] Documentation has been added / updated
- [x ] Changes have been communicated to collaborators

### Does this PR introduce breaking changes? What changes might users need to make to their code?

No.

### Does this PR change default behavior?

Yes - this will change the ZDC geometry used by default.

### CODE REVIEW

**Review by veprbl — DISMISSED** [review]

File: compact/far_forward/roman_pots_eRD24_design.xml [hunk 1]
```diff
@@ -36,24 +36,27 @@
...
 
+    	<!--
+
 	<constant name="ForwardRomanPotStation1_insertion_intermediate" value="3.2*cm + 0.6*cm"/>
     <constant name="ForwardRomanPotStation2_insertion_intermediate" value="3.2*cm + 0.6*cm"/>
 
     <constant name="ForwardRomanPotStation1_insertion_central" value="3.2*cm + 0.7*cm"/>
     <constant name="ForwardRomanPotStation2_insertion_central" value="3.2*cm + 0.7*cm"/>
 
+	--!>
```
Comment (veprbl) [inline compact/far_forward/roman_pots_eRD24_design.xml:59]:
```suggestion
	-->
```
https://github.com/eic/epic/pull/417/files

File: compact/far_forward_arches.xml [hunk 2]
```diff
@@ -8,7 +8,7 @@
   <include ref="far_forward/B0_tracker.xml"/>
   <include ref="far_forward/B0_ECal.xml"/>
   <include ref="far_forward/offM_tracker.xml"/>
-  <include ref="far_forward/ZDC_Athena.xml"/>
+  <include ref="far_forward/ZDC.xml"/>
```
Comment (veprbl) [inline compact/far_forward_arches.xml:11]: We could consider moving this stuff to https://github.com/eic/epic/blob/main/templates/epic.xml.jinja2 and add some configutation conditionals there.

Comment (ajentsch) [reply compact/far_forward_arches.xml:11]: I am not sure that is necessary - the "ZDC.xml" file is the baseline option.

Comment (veprbl) [reply compact/far_forward_arches.xml:11]: Okay.

**Review by veprbl — APPROVED** [review]

**Pull request merged**